src/Context.php

```diff
@@ -95,6 +95,31 @@ public static function parse_utm_campaign_param(string $cf_local_project): strin
         return '';
     }
 
+    /**
+     * Set p4_blocks datalayer value
+     *
+     * @param array  $context Context to be set.
+     * @param object $post That the context refers to.
+     */
+    public static function set_p4_blocks_datalayer(array &$context, object $post): void
+    {
+        preg_match_all('/wp:planet4-blocks\/(\S+)|wp:gravityforms\/(\S+)*/', $post->post_content, $matches);
```

Comment:
This regex matches all the occurrences of `wp:planet4-blocks/` or `wp:gravityforms/` until the next empty space. 

For example, 
`wp:planet4-blocks/columns`, `wp:planet4-blocks/cover`, `wp:gravityforms/form`